Assets/Resources/JSON/GiftItems.json

```diff
@@ -0,0 +1,28 @@
+{
+  "gifts" : [
+    {
+      "giftName" : "5 Dog Food Cans",
+      "value" : 5,
+      "valueCurrencyType" : 0
```

Comment:
I think this would read better as a string we parse to an enum: https://msdn.microsoft.com/en-us/library/essfb559(v=vs.110).aspx